data/dif10_section_titles.csv

```diff
@@ -2,7 +2,6 @@ dif10_section_title
 Summary
```

Comment:
All top level field which has children, check if need to add field as simple array